pkg/configs/api/api.go

```diff
@@ -93,12 +97,22 @@ func (a *API) getConfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Header().Set("Content-Type", "application/json")
-	if err := json.NewEncoder(w).Encode(cfg); err != nil {
+	switch ParseConfigFormat(r.Header.Get("Accept"), FormatJSON) {
+	case FormatJSON:
+		w.Header().Set("Content-Type", "application/json")
+		err = json.NewEncoder(w).Encode(cfg)
+	case FormatYAML:
+		w.Header().Set("Content-Type", "application/yaml")
+		err = yaml.NewEncoder(w).Encode(cfg)
+	default:
+		// should not be here
+		level.Error(logger).Log("msg", "unknown Format")
```

Comment:
Not sure it's worth to log it as error. This case depends on the client input and we generally don't log as error any `4xx` error. We could omit this log at all.